src/meta_schedule/profiler.cc

```diff
@@ -79,7 +80,8 @@ PackedFunc ProfilerTimedScope(String name) {
                                     tik = std::chrono::high_resolution_clock::now(),  //
                                     name = std::move(name)]() {
       auto tok = std::chrono::high_resolution_clock::now();
-      double duration = std::chrono::duration_cast<std::chrono::seconds>(tok - tik).count();
+      double duration =
+          std::chrono::duration_cast<std::chrono::nanoseconds>(tok - tik).count() / 1e9;
```

Comment:
quick question: what's the diff between the two?

Reply:
Turns out the previous one would create some inaccuracy in time counting, thus impacting the final result.

Reply:
Thanks for sharing!